### DISCUSSION TITLE
Priorities in policy "Dissolution and Asset Distribution"

### DISCUSSION THREAD

**Discussion by Sanne** [commonhaus/foundation#68] [Q&A]
The current draft for [Dissolution and Asset Distribution](https://github.com/commonhaus/foundation-draft/blob/main/policies/dissolution-policy.md#dissolution-and-asset-distribution) mentions these two steps to be executed in strict order:

1. Settling Liabilities: First, we will pay off any outstanding debts or obligations the Foundation has incurred.
2. Asset Distribution: After clearing all liabilities, any remaining assets will be distributed to one or more organizations that share a similar mission or goals as the CF. 

I appreciate the need of paying off any outstanding debts, however it concerns me that the IP such as OSS project trademarks would end up being handed over to whomewer the Foundation still owes something.

Clearly the intention is for such projects and trademarks to be moved to a different non-profit with aligned goals, but in practice I fear this wording would lead for some big cloud provider e.g. AWS to own them all as soon as someone introduces a little bug in their devops scripts which decide how many CI nodes to start...

Could this be clarified? Would it even be legal to state that such IP will never be sold to cover debts, or be transferred to a different non-profit at higher priority?

**Comment by kenfinnigan** [comment]
Instead of the next sentence for (2) being `This could include other nonprofit groups, like those classified under Section 501(c)(3) (charitable organizations) or other 501(c)(6) organizations (business leagues or chambers of commerce).` would it be preferable, if it's legal to do so, for it to explicitly require them going to a charitable org as opposed to only being "could include"?

**Reply by Sanne** [reply]
No sorry that doesn't address my concern; imagine this (far fetched, hypotethical) scenario:

Someone in one of the projects makes a significant mistake in CI management scripts, incurring a huge bill on AWS. The CF goes bust as they can't pay the bill.. at this point CF has a debt to AWS and rule _1_ is applied first: assets get liquidated so to repay debts as much as possible; this implies IP and trademarks need to get sold on the open market to the best bidder.

Rule _2_ gets applied subsequently.. but it's too late, the IP assets are gone.

**Comment by ebullient** [comment]
I was talking w/ Gavin about this today. Part of me was going to nix this policy altogether (in favor of the default from Florida law), but it sounds like things need to be clearer, so.. I guess we'll keep it.

I would love for folks to take a crack at making this clearer w.r.t. OSS projects. Some of this is filed under the "be responsible stewards and take fiduciary duties seriously" umbrella.

- https://www.councilofnonprofits.org/running-nonprofit/governance-leadership/dissolving-nonprofit-corporation
- https://www.flsenate.gov/Laws/Statutes/2020/0617.1421

As I understand them, there are constraints around assets donated to a non-profit: they can not be "sold" back to a for-profit entity. I think it is in our interest to define what constitutes a project (source code, domains, project-specific logos and trademarks, .. anything else?), and indicate that if the foundation were to dissolve, each project would be able to choose the destination that meets their needs.  By law, it must be another non-profit entity (as I understand it). But each project should have their choice. Several could choose together, if they're similar enough, and that all works out.

Dealing with unpaid debt is a serious issue, best dealt with by not getting into serious debt to begin with (which is why I'm not promising the moon with infrastructure or cloud credits or staff or .. )

**Reply by Sanne** [reply]
> there are constraints around assets donated to a non-profit: they can not be "sold" back to a for-profit entity

This sounds very promising - but I'll assume it's restricted to certain types of assets? e.g. if we donate money I'd assume the CF would be able to spend it on cloud compute.. so there must be something more to this.

Also, how is this possible in practice? It suggests that once one of our trademarks is donated there is no way that it will ever be able to be used ever again in a commercial setting? I do like the idea, but I have an hard time believing that this would be possible w/o exception, and I suspect that creditors in a case of bankrupcy might be able to push for such exceptions. (I'm not versed at all in such matters, I'm just assuming this would be plausible IMO).

> Dealing with unpaid debt is a serious issue, best dealt with by not getting into serious debt to begin with

I agree and I accept my example is peculiar. Still in presence of many projects, many developers eager to explore things, and I assume minimal viable funds it's not that far fetched, and I'd prefer it if the IP assets of projects were protected from vanishing suddenly because of some mistake.
It would be especially regrettable if one project screws up and takes down all others with it - the mere risk of something like this possibly happening would make us more suspicious of how other projects within CF operate, in turn leading to a desire of influencing other projects, adding additional rules and processes to try prevent foolishness.

I feel this risk needs to be strictly mitigated so as to protect the goal of each project to have significant independence.

**Reply by ebullient** [reply]
"Also, how is this possible in practice? It suggests that once one of our trademarks is donated there is no way that it will ever be able to be used ever again in a commercial setting?"

Yes. That is what it means. It can never be used by a for-profit entity. It is why donating to a foundation is powerful.

More inspiration here: 
https://nonprofitquarterly.org/nonprofit-dissolution-what-to-do-when-closing-the-doors/

> Most states require that an organization’s assets be distributed to other charitable organizations or governmental bodies. These laws ensure that assets amassed for charitable or other nonprofit activities continue to be used for similar purposes.

That is really what it means. The donated IP can't ever come "home" .. ever. But it can find a new-next home at another non-profit, if the worst happens.

You aren't wrong here, Sanne. I think we should edit that document to identify what a project is, and state that if things must dissolve, projects are treated as entities, and each (with EGC/CFC blessing?) decides where to go. Some "in good faith" words probably belong here.

If we want to hit the corner cases, then there are things to address re: negligence.

ALSO, as a sidebar, that policy should really move into the bylaws (probably included with liability/indemnification). Let's fix first, then move it.

**Reply by gavinking** [reply]
>  I think it is in our interest to define what constitutes a project (source code, domains, project-specific logos and trademarks, .. anything else?)

This is a _very_ good idea.

**Reply by Sanne** [reply]
I like this idea but I doubt that our own made rules and wishes would stand in the face of a case of bankrupcy. It would be like defaulting on a home mortgage and claiming you're not leaving the house as you're grown emotionally attached to it, and your family's rules don't allow you to vacate the house.

I'm really not an expert in this matter but the current proposal makes me nervous, as assets such as our IP would be quite defenseless in such a situation, and CF being small and I assume with little/minimal viable financial backing it's going to be stressing.

Would some insurance policy be possible?  I hate to bring it up as it would require money, which we were hoping to avoid, but I don't have better ideas.

**Reply by ebullient** [reply]
Yes, we have insurance.

**Comment by gavinking** [comment]
> I appreciate the need of paying off any outstanding debts, however it concerns me that the IP such as OSS project trademarks would end up being handed over to whomewer the Foundation still owes something.

I think there's not much we can do about this without the help of a lawyer. I don't believe we can just write some policy and think that it's going to shield some assets of the foundation during bankruptcy proceeding. Perhaps there's _something_ that can be done but need a lawyer for that.

**Reply by Sanne** [reply]
I agree.
I also appreciate that there's some fundamental unavoidble risk, no matter how we spin it - but there's certainly ways to make the worse outcomes less likely.
For example, what would happen if our current IP owner would go bust? Pretty much the same situation, except its financial situation makes such a scenario far less of a concern.

**Reply by ebullient** [reply]
I have a lawyer, and will ask.. we can still make a statement about what we would like to attempt first. It makes for an easier conversation. Is this possible? If you look at requirements of law, these things must be distributed to other government or non-profit entities. I believe that we can say that minimally, project assets be kept together.

Dissolution can happen for a more than just financial reasons. 

This resource (https://nonprofitquarterly.org/nonprofit-dissolution-what-to-do-when-closing-the-doors/) really does have some good ideas in it. I was wrestling with trademark trolls yesterday, and I'm not sure I'm happy with where that landed. Stupid words.

**Comment by ebullient** [comment]
I took a crack at something here: #75 

As we usually do, I'm going to close this discussion as outdated (in favor of the PR). The content will still be here for reference.
